package.json

```diff
@@ -0,0 +1,32 @@
+{
+  "name": "matterfront",
+  "version": "0.1.0",
+  "description": "Mattermost frontend app powered by electron",
+  "main": "./src/main.js",
+  "devDependencies": {
+    "electron-packager": "^5.1.1",
+    "electron-prebuilt": "^0.34.2"
+  },
+  "electronVersion":"0.34.1",
+  "scripts": {
+    "start":"electron ./",
```

Comment:
nah, the `src` file is pretty common, even in electron apps. You did just fine as far as the node idioms go. :smile: 

I'm working through a heirarchical config system for `build` and `run` type operations that I'll submit a PR for either tonight or tomorrow... which will also make all that stuff *way* easier to reason about from within the `src` itself.  You'll be able to just make a reference to the app's config, which will be reasoned about based on how it was built *and* launched. (Going to do some cool stuff with [nconf](https://www.npmjs.com/package/nconf)